browser/lib/transport/xhrrequest.js

```diff
@@ -115,7 +115,7 @@ var XHRRequest = (function() {
 
 		if(body) {
 			var contentType = headers['content-type'] || (headers['content-type'] = 'application/json');
-			if(contentType == 'application/json' && typeof(body) != 'string')
```

Comment:
`contentType` on this line is the content type of the body that the client is POSTing, so is set by us (ie ably-js) -- at this stage the request hasn't actually started yet. So there's no opportunity for it to be anything other than what we set it to, so this change isn't necessary.